server/lib/chat/chat-service.ts

```diff
@@ -573,6 +575,39 @@ export default class ChatService {
     }
   }
 
+  async searchChannels({
+    userId,
+    limit,
+    pageNumber,
+    searchStr,
+  }: {
+    userId: SbUserId
+    limit: number
+    pageNumber: number
+    searchStr?: string
+  }): Promise<SearchChannelsResponse> {
+    const channels = await searchChannels({
```

Comment:
It doesn't really seem like there's another use for this model method, so why not just do this with a join and return the channels + the UserChannelEntry's in one go?